Cargo.toml

```diff
@@ -52,6 +52,7 @@ pallet-burner = { default-features = false, path = "pallets/burner" }
 pallet-payments = { default-features = false, path = "pallets/payments" }
 pallet-communities = { default-features = false, path = "pallets/communities" }
 pallet-communities-manager = { default-features = false, path = "pallets/communities-manager" }
+pallet-fee-handler = { default-features = false, path = "pallets/fee-handler" }
```

Comment:
Is this a WIP? It'd be nicer to have it as a separate PR. This way, we can immediately approve the _"hardcoded"_ solution after requested changes, and start reviewing the extension pallet later.

Reply:
Nvm. Noticed this is different from #380